src/agent0/core/hyperdrive/utilities/run_bots/execute_multi_agent_trades.py

```diff
@@ -237,7 +237,11 @@ async def _async_match_contract_call_to_trade(
 
         case HyperdriveActionType.OPEN_LONG:
             trade_result = await interface.async_open_long(
-                agent, trade.trade_amount, trade.slippage_tolerance, nonce=nonce
+                agent,
+                trade.trade_amount,
+                trade.slippage_tolerance,
+                nonce=nonce,
+                gas_limit=trade.gas_limit,
```

Comment:
I think we intended for these to be flipped, so the order is `slippage, gas_limit, nonce`.

Reply:
moved nonce to the end and also added keywords in 521ea3e4c9f233cb8e22bdfc9db29042702e9980